### ISSUE TITLE
Ensure that for all models, the evaluation method is correct

### ISSUE THREAD

**Issue by mr-carlitos** [theoliver7/uzh-aml#8]
For the evaluation we propose the following procedure such that reproducibility is assured.

- Randomly split each dataset into two parts: 90% as training set and the remaining 10% as test set and save the ids.
- We will perform cross-validation on the train data with a split of 5.
- Report the performance.
